1619973240890 [handle_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Arm_moveit.py", line 93, in handle_command
    self.group.set_pose_target(self.target, end_link)
AttributeError: moveit_handle instance has no attribute 'target'
] 
1619973240899 [Finish initializing arm_moveit] 
1619973270900 [Position received] 
1619973271901 [command: %f] [2] 
1619973288891 [Position received] 
1619973289892 [command: %f] [2] 
1619973335693 [Position received] 
1619973336694 [command: %f] [2] 
1619973367294 [Position received] 
1619973368295 [command: %f] [2] 
1619973398295 [Position received] 
1619973399297 [command: %f] [2] 
1619973577092 [Position received] 
1619973578093 [command: %f] [2] 
